reportportal/values.yaml

```diff
@@ -36,7 +36,9 @@ uat:
   sessionLiveTime: 86400
   podAnnotations: {}
   jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
+  additionalEnvironments: {}
```

Comment:
Please add comment on the value, what exactly it does. Ideally our values should look like here: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
(each value commented, examples provided)